### ISSUE TITLE
Update parsing

### ISSUE THREAD

**Issue by jamesdbrock** [purescript/registry#108]
```json
{
  "name": "parsing-dataview",
  "ref": "v3.2.1",
  "compiler": "0.15.4"
}
```

**Comment by jamesdbrock** [comment]
```json
{
  "name": "parsing-dataview",
  "ref": "v3.2.2",
  "compiler": "0.15.4"
}
```

**Comment by jamesdbrock** [comment]
I want to transfer this repo to rowtype-yoga. Therefore I need a `purs.json` manifest with myself as an owner. But I don't want to hand-maintain all the `dependencies` in the manifest, I want the Registry to solve the `dependencies` for me. How?

**Comment by f-f** [comment]
Yeah, I would recomment avoiding `purs.json` files until the alpha of the new Spago is out.

**Comment by jamesdbrock** [comment]
> The easy thing to do is remove the purs.json file and just transfer the repo over on GitHub; there’s a daily Action that looks for GitHub transfers

Ok cool, I deleted the `purs.json`, and the repo has already been transferred to https://github.com/rowtype-yoga/purescript-parsing-dataview

So I’ll watch this file and within 24 hours I should see transfer reflected here?

https://github.com/purescript/registry-index/blob/main/pa/rs/parsing-dataview

**Comment by thomashoneyman** [comment]
You’d see it in the metadata:
https://github.com/purescript/registry/tree/main/metadata

**Comment by jamesdbrock** [comment]
https://github.com/purescript/registry/blob/main/metadata/parsing-dataview.json

**Comment by thomashoneyman** [comment]
Sorry, one other thing @jamesdbrock -- the transferring happens when the registry tries to publish your package but sees it is in a new location. So you will see the transfer reflected in the metadata file within 24 hours of publishing a new tag from the new location.

**Comment by jamesdbrock** [comment]
```json
{
  "name": "parsing-dataview",
  "ref": "v3.2.4",
  "compiler": "0.15.4"
}
```
